xdsl/xdsl_opt_main.py

```diff
@@ -61,7 +63,7 @@ class xDSLOptMain:
     stream.
     """
 
-    pipeline: List[tuple[str, Callable[[ModuleOp], None]]]
+    pipeline: List[ModulePass]
```

Comment:
Oh no, the pipeline already infected xDSL :fearful:

Reply:
Ahahah good catch! It's just a list of passes for now, so I don't mind renaming it `passes`.

Reply:
I don't mind, it was there before, so it can stay (for the time being)...

Reply:
Oh, forgot it was already there and named in functions and everything.. I'd better leave it as is for this PR then!